code/__DEFINES/logging.dm

```diff
@@ -3,6 +3,7 @@
 #define INVESTIGATE_ATMOS "atmos"
 #define INVESTIGATE_BOTANY "botany"
 #define INVESTIGATE_CARGO "cargo"
+#define INVESTIGATE_CIRCUIT "circuit"
```

Comment:
Props for adding proper circuit logging

Reply:
God I hope I've got it right though! I wouldn't guarantee it.